x-pack/plugins/security_solution/public/resolver/store/data/reducer.test.ts

```diff
@@ -40,6 +73,15 @@ describe('Resolver Data Middleware', () => {
           },
         };
         store.dispatch(action);
+        const relatedAction: DataAction = {
```

Comment:
The `describe` above doesn't say anything about having related events. Also, some of these specs worked fine w/o your changes. Consider structuring the spec like:

```
describe('when data was received and the ancestry and children edges had cursors', () => {
  beforeEach(() => {
    // whatever was here before
  })
  // old specs. e.g. `it('should indicate there are additional ancestor', () => {`
  describe('and when related events were returned', () => {
    beforeEach(() => {
      // your new logic.
    })
})
```
The second, nested, `describe` could help the reader to understand the purpose of the new logic in the `beforeEach`.

Reply:
https://github.com/elastic/kibana/pull/70097/commits/a9ba1d473bbd7e2fe3c98ac7ada0ba46f60b79f0